### DISCUSSION TITLE
Code Changes

### DISCUSSION THREAD

**Discussion by aljawaid** [JustFxDev/ThemeMaestro#10] [General]
Hello, I see you are making changes to the code.... if you cant create the release yourself, comment here whenever youre ready and what version number you want and I will create the release for you.  Just remember, if you make any changes after the release, you will have to create a further new release... My point is, there is no edit release option.... well there is, but it can get complicated.

**Comment by JustFxDev** [comment]
Hi aljaweid,

yes i did some minor changes. Mostly syntax errors oder readme.md improvements. I will not change a release afterwards. I think is not a good approach to maintain code / software.

If i have enough new changes together i will create a new release (maybe with your assistance). For the moment it is not planned to change anything in that direction bevor that happened.

Thanks again for your help!

Cheers, Fx
